### ISSUE TITLE
Set default panel tab to Knobs

### ISSUE THREAD

**Issue by scottleblang** [storybooks/storybook#6739]
I am using Storybook for React and make heavy use of Knobs. In all instances, in the bottom panel, the default tab is the Actions panel. I would like to request the option to:
1)  Set the option for a Default panel to be selected - i.e. Knobs rather than Actions
2) Option to remove other tabs - i.e. Actions, Story, etc.

As far as I can tell, these options don't exist. 

Thanks!

**Comment by romanrandom** [comment]
Also reordering the imports in `addons.js` does the trick

**Comment by Rohit1508** [comment]
Any update on this, how to make the panel default

**Comment by zero-t4** [comment]
> Any update on this, how to make the panel default

Just re-order your `addons` in `.storybook/main.js` 

```
// ...
addons: [
    '@storybook/addon-knobs/register',
    '@storybook/addon-actions/register',
    '@storybook/addon-backgrounds/register'
  ],
// ...
```

**Comment by shilman** [comment]
FYI, we’ve released addon-controls in 6.0-beta. Controls are portable, auto-generated knobs that are intended to replace addon-knobs long term.

- [Controls README](https://github.com/storybookjs/storybook/tree/next/addons/controls)
- [Migration instructions](https://github.com/storybookjs/storybook/blob/next/addons/controls/README.md#how-do-i-migrate-from-addon-knobs)
- [Walkthrough: Controls w/ CRA & TypeScript](https://gist.github.com/shilman/69c1dd41a466bae137cc88bd2c6ef487)

Please upgrade and try them out today!
